src/browserHelperFile.json

```diff
@@ -0,0 +1 @@
|2","chrome/83.0.4103.89|1","chrome/75.0.3770.142|2","chrome/38.0.1025.166|1","chrome/87.0.4280.148|2","firefox/60.0|2","chrome/80.0.3987.106|2","chrome/65.0.3325.181|2","chrome/61.0.3163.100|2","chrome/74.0.3729.108|2","chrome/81.0.4044.138|2","chrome/47.0.2526.106|2","chrome/71.0.3700.0|2","chrome/65.0.3325.109|2","chrome/79.0.3945.130|1","chrome/81.0.4044.142|2","chrome/88.0.4324.208|1","chrome/76.0.3809.132|2","chrome/89.0.4389.82|1","chrome/79.0.3945.136|2","firefox/83.0|2","chrome/90.0.4421.0|2","chrome/88.0.4315.7|2","chrome/88.0.4322.2|2","chrome/88.0.4320.0|2","chrome/59.0.3071.125|2","chrome/73.0.3683.75|1","chrome/90.0.4427.0|2","chrome/91.0.4438.4|2","chrome/74.0.3729.169|2","chrome/90.0.4403.0|2","firefox/72.0|2","firefox/73.0|2","chrome/91.0.4438.0|2","chrome/72.0.3750.0|2","firefox/38.0|2","chrome/88.0.4324.93|2","chrome/88.0.4324.96|1","chrome/47.0.2526.111|2","safari/605.1.15|1","chrome/79.0.3945.88|2","chrome/51.0.2704.84|2","safari/604.3.5|2","chrome/85.0.4183.101|2","chrome/77.0.3865.75|2","firefox/52.0|1","chrome/69.0.3497.100|2","chrome/75.0.3770.143|2","firefox/88.0|2","chrome/88.0.4302.0|2","chrome/88.0.4318.1|2","chrome/88.0.4310.3|2","chrome/88.0.4313.0|2","chrome/89.0.0.0|2","chrome/56.0.2924.87|2","firefox/65.0|2","chrome/71.0.3578.99|2","chrome/89.0.4385.0|2","chrome/83.0.4103.97|2","firefox/45.0|2","chrome/80.0.3987.162|2","chrome/78.0.3904.87|2","chrome/63.0.3239.132|2","chrome/88.0.0.0|2","chrome/84.0.4147.125|1","chrome/80.0.3987.145|1","chrome/44.0.2403.119|2","chrome/84.0.4147.89|1","chrome/86.0.4240.193|2","chrome/79.0.3945.120|2","chrome/49.0.2623.112|2","chrome/41.0.2272.96|2","firefox/82.0|2","chrome/99.0.3538.77|2","chrome/88.0.4324.146|1","chrome/88.0.4324.104|1","chrome/84.0.4147.105|2","chrome/88.0.4324.41|2","chrome/88.0.4291.1|2","firefox/48.0|2","chrome/88.0.4324.109|2","chrome/73.0.3683.75|2","chrome/48.0.2564.97|2","chrome/89.0.4389.76|2","chrome/72.0.3626.121|2","chrome/91.0.4440.0|2","chrome/84.0.4147.110|2","firefox/71.0|2","chrome/90.0.4405.0|2","firefox/57.0|2","chrome/67.0.3396.87|2","chrome/88.0.4324.153|2","firefox/77.0|2","safari/604.4.7|2","chrome/80.0.3987.122|1","firefox/56.0|1","chrome/89.0.4389.86|2","firefox/75.0|2","chrome/88.0.4298.0|2","chrome/74.0.3729.172|2","chrome/87.0.4280.107|2","chrome/88.0.4308.0|2","chrome/88.0.4313.1|2","chrome/88.0.4324.79|2","chrome/88.0.4307.2|2","chrome/89.0.4371.0|2","chrome/87.0.4280.86|2","firefox/80.0|2","chrome/1.0.154.50|2","chrome/74.0.3729.157|2","chrome/70.0.3538.25|2","chrome/63.0.3239.84|2","chrome/80.0.3987.132|1","chrome/72.0.3626.81|2","chrome/88.8.383.407|2","chrome/86.0.4240.197|2","chrome/86.0.4240.75|1","chrome/87.0.4280.88|1","chrome/58.0.3029.83|2","firefox/64.0|2","firefox/78.0|1","chrome/89.0.4389.57|2","chrome/88.0.4288.0|2","chrome/11.0.696.71|2","firefox/3.6.10|2","chrome/88.0.4324.155|2","chrome/66.0.3359.158|2","firefox/46.0|2","chrome/48.0.2564.109|2","chrome/79.0.3945.116|2","chrome/90.0.4430.19|2","chrome/62.0.3202.94|2"]
```

Comment:
I think we should stick for one file naming convention. Could you please use the `-` separator. Is the camelCase some json naming convention?

Reply:
Used it to obviously distinguish the data files from the code files. But will use the - separator if you think it is better that way.

Reply:
I would place them in a subfolder and used the separator.